### DISCUSSION TITLE
Why some repository readme shows a sticky button to show TOC of the readme content but most repository don’t have

### DISCUSSION THREAD

**Discussion by jwenjian** [community/community#23965] [New to GitHub]
<p>See <a href="https://github.com/rails/rails" class="inline-onebox" rel="noopener nofollow ugc">GitHub - rails/rails: Ruby on Rails</a> for the button</p>
<p>How do I get this button for my repo ？</p>

**Comment by ylemkimon** [comment, marked as answer]
<p>There is a <code>Table of contents</code> feature in the repo settings:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/7/7/77612c59c4d1d819b3c7fc56003b8803a93222cc.png" data-download-href="/uploads/short-url/h250IYtCLvC5HR5up5SLv1TopiY.png?dl=1" title="2021-03-26 (1)" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/888148/181080430-9c0b6392-1dad-4e08-ac2c-f306d0a293d9.png" alt="2021-03-26 (1)" data-base62-sha1="h250IYtCLvC5HR5up5SLv1TopiY" width="690" height="62" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/7/77612c59c4d1d819b3c7fc56003b8803a93222cc_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">2021-03-26 (1)</span><span class="informations">913×83 4.19 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>However, it seems it’s on the beta, i.e., not generally available yet.</p>

**Comment by jwenjian** [comment]
<p>Thanks, I thought that was a hidden feature…… seems like I’m not in the beta plan :smile:</p>

**Comment by jwenjian** [comment]
<p>Seems the feature is opened for every one</p>
